tox.ini

```diff
@@ -70,6 +71,26 @@ commands =
 
     pytest python
 
+[testenv:jax-tests]
+# this environement runs tests for the jax bindings
+deps =
+    wheel
+    cmake
+
+    numpy
+    jax
+    pytest
+
+
+
+changedir = sphericart-jax
+passenv=
+    PIP_EXTRA_INDEX_URL
```

Comment:
You should not need this, it is about allowing users to install a CPU-only torch package when running on Linux with a CUDA compiler.